scripts/htmlDependencies.R

```diff
@@ -11,7 +11,7 @@ versions <- list()
 message("Installing GitHub packages: bslib, shiny, htmltools")
 withr::local_temp_libpaths()
 ignore <- capture.output({
-  pak::pkg_install(c("rstudio/bslib", "cran::shiny", "cran::htmltools"))
+  pak::pkg_install(c("rstudio/bslib@v0.6.1-plus-task-button", "rstudio/shiny@main", "cran::htmltools"))
```

Comment:
Sorry about missing this! I think we should move away from allowing `@main` to be the ref here. Instead, if we're not using the version on CRAN, I propose we have a moving `py-shiny` tag in these repos and that we update those tags and before running this script.

Note that the actual ref used is stored in a JSON file when the dependencies are updated.

```suggestion
  pak::pkg_install(c("rstudio/bslib@py-shiny", "rstudio/shiny@py-shiny", "cran::htmltools"))
```

I'll add a tag for shiny and bslib. I'll point shiny at HEAD on main and I'll point bslib's tag at the previously used version of bslib so you can start from there.

Reply:
I guess there's a chicken-and-egg problem here: If we update the tag in bslib before merging in py-shiny, then for a period the tag is out of date. If we want to be careful, as in this PR, we can start the PR with a branch or short ref and then, before merging move the tag and update this script to use the new tag.

Reply:
I'm taking the careful approach. 👍